core/build.gradle.kts

```diff
@@ -383,8 +383,10 @@ tasks.withType<AbstractDokkaLeafTask>().configureEach {
     pluginsMapConfiguration.set(mapOf("org.jetbrains.dokka.base.DokkaBase" to """{ "templatesDir" : "${projectDir.toString().replace('\\', '/')}/dokka-templates" }"""))
 
     failOnWarning.set(true)
+    dokkaSourceSets.named("commonMain") {
+        samples.from(kotlin.sourceSets.commonTest.map { it.kotlin.srcDirs })
+    }
```

Comment:
Even though we don't have them now, we could add samples to our platform-specific declarations as well. Can this ability be preserved without specifying every source set explicitly?

Reply:
Yeah, sure! I will update the PR!